how-to/software/upgrade-your-release.md

```diff
@@ -7,10 +7,24 @@ We recommend running a Long Term Support (LTS) release as it provides 5 years of
 
 After the initial standard support period ends for an LTS release, an extended maintenance period is available via an [Ubuntu Pro subscription](http://ubuntu.com/pro), which provides coverage for an additional five years and is available for free on up to five machines. Find out more about the [release lifecycle and support period](https://ubuntu.com/about/release-cycle) for your release.
 
+## Update vs Upgrade vs Release-Upgrade
+
+```{note}
+In this guide, we use the term "upgrade" in two differente contexts:
+ * In the **pre-upgrade checklist**, **upgrade** refers to updating and upgrading the currently installed packages within your existing Ubuntu release.
+ * In the **upgrade the system** section, **upgrade** refers to performing a distribution upgrade to a new Ubuntu release using `do-release-upgrade`.
+```
+| Command | Purpose |
+| :--- | :---: |
+| apt update | Updates packages index (metadata) | 
+| apt upgrade | Upgrades only installed packages within the current distribution |
+| do-release-upgrade | Upgrades the entire OS to next release |
```

Comment:
while explaining these differences is not a bad idea, I'd rather reformulate or clarify the sections (pre-upgrade checklist, upgrade the system).

Reply:
sorry, didn't see that in your bug, alright then :)